rust/src/cli/persist_nic.rs

```diff
@@ -110,8 +112,10 @@ pub(crate) fn run_persist_immediately(
             iface.name()
         );
         if !dry_run {
+            let iface_name = iface.name();
             changed |=
-                persist_iface_name_via_systemd_link(root, mac, iface.name())?;
+                persist_iface_name_via_systemd_link(root, mac, iface_name)?;
+            kargs.push(format!("ifname={iface_name}:{mac}"));
```

Comment:
Since we have this in two places, maybe worth a `fn format_ifname_karg(iface_name: &str, mac: &str) -> String` ?